test/cookbooks/test_helper/recipes/setup_waiver_fixture.rb

```diff
@@ -0,0 +1,19 @@
+
+[
+  "controls",
+  "waivers",
+].each do |subdir|
+  directory "#{node["audit"]["profiles"]["waiver-test-profile"]["path"]}/#{subdir}" do
```

Comment:
`dig` is probably good to use on this type of code.

```ruby 
node.dig "audit", "profiles", "waiver-test-profile", "path"
```

Reply:
This is a Chef Infra node object, and it handles auto-vivification. The syntax I used is more traditional for Chef Infra code.